app/views/finance/balancing/confirm.html.haml

```diff
@@ -4,6 +4,17 @@
     %p
       %b= heading_helper FinancialTransaction, :financial_transaction_type
       = select_tag :type, options_for_select(FinancialTransactionType.order(:name).map { |t| [ t.name, t.id ] })
+  - if FinancialTransaction.where(ordergroup: nil).any?
```

Comment:
what do you think about merging both `if` into `if FoodsoftConfig[:use_financial_links]`? I think the check for an "foodcoop transaction" is somehow outdated now (applies to `new_collection.html.haml` too). I would also prefer to make both checkboxes checked by default, since that's the common case. we can do that also in a follow up commit, if you don't want to include it here.

Reply:
I was already thinking about adding another config option "use_foodcoop_transactions", what do you think about that?
Or one option "use double-entry accounting" for both financial links and foodcoop transactions?

Reply:
i'd prefer to keep the number of options low. IMHO showing `user_financial_links` as _Use financial links (usable for "double-entry accounting")_ should be enough.